### PR TITLE
Add class UBERON:8410004 jejuno-ileal junction

### PR DESCRIPTION
**Pull request by paolaroncaglia**
Addresses #1662

### CODE REVIEW

**Review by dosumis — CHANGES_REQUESTED** [review]
publication xrefs -> def xrefs

**Comment by paolaroncaglia** [comment]
@dosumis 

> publication xrefs -> def xrefs

Done, thanks.

File: uberon_edit.obo [hunk 1]
```diff
@@ -195227,6 +195227,20 @@ relationship: dc-creator http://orcid.org/0000-0003-2034-601X
 relationship: part_of UBERON:0004647 ! liver lobule
 property_value: http://purl.org/dc/elements/1.1/date 2020-10-05T15:52:22Z xsd:dateTime
 
+[Term]
+id: UBERON:8410004
+name: jejuno-ileal junction
+def: "The anatomical junction between the jejunum and the ileum part of the intestine. Although there is no clear distinct section, the ileum is narrower and has smaller villi than the jejunum." [PMID:21802415]
+synonym: "jejunoileal junction" EXACT []
+xref: https://doi.org/10.1016/B978-1-4377-2206-2.00068-3
```
Comment (dosumis) [inline uberon_edit.obo:195235]: Please move to def xrefs as DOI:10.1016/B978-1-4377-2206-2.00068-3

**Review by dosumis — CHANGES_REQUESTED** [review]
see comment on DOI

**Pull request closed without merging**